src/Yaapii.Xambly/Directive/Directives.cs

```diff
@@ -525,32 +524,40 @@ public Directives Up()
         {
             this.all.Add(new UpDirective());
             return this;
-        }
-
-        /// <summary>
-        /// Go to xpath.
+        }
+
+        /// <summary>
+        /// Moves cursor to the nodes found by XPath.
         /// </summary>
-        /// <param name="path">path to go to</param>
-        /// <exception cref="IllegalArgumentException"/>
-        /// <returns>This object</returns>
-        public Directives Xpath(Object path)
+        /// <param name="path">XPath</param>
+        /// <param name="rootDefNamespacePrefix">Default namespace prefix for use in XPath or empty string</param>
+        /// <param name="defNamesapceAndPrefixDictionary">Optional tupel defining default namespace prefixes from children nodes. Always write multiples of two: Namespace and prefix</param>
+        public Directives Xpath(object path, string rootDefNamespacePrefix = "", params string[] defNamesapceAndPrefixDictionary)
```

Comment:
The name "defNamesapceAndPrefixDictionary" is not understandable, especially when it is a string array that you inject. Names of variables must never be a type of an object but what it IS.
"Dictionary" is a type.